### PR TITLE
[chore] add docs for oras_login --resolve flag

### LINKED ISSUE #200: Add example for "--resolve" flag
### What is the version of your ORAS CLI

v1.0.0

### What would you like to be added?

There is a flag `--resolve host:port:address[:address_port]`  which is used to customize DNS for the registry, formatted in `host:port:address[:address_port]`. We need to add this example to ORAS commands with `--resolve` flag.

Giving an example:

```
oras login prod.registry.io:80 --resolve prod.registry.io:80:127.0.0.1:5000 --plain-http
```

### Why is this needed for ORAS?

Let users understand how to use the `--resolve` flag 

### Are you willing to submit PRs to contribute to this feature?

- [ ] Yes, I am willing to implement it.

### PR DESCRIPTION
**Pull request by amands98**
This PR fixes #200

### CODE REVIEW

File: docs/commands/oras_login.mdx [hunk 1]
```diff
@@ -49,6 +49,12 @@ Log in with username and password in an interactive terminal and no TLS check:
 oras login --insecure localhost:5000
 ```
 
+Log in to registry with customized DNS resolution
+
+```bash
+oras login prod.registry.io:80 --resolve prod.registry.io:80:127.0.0.1:5000 --plain-http
```
Comment (qweeah) [inline docs/commands/oras_login.mdx:55]: Below command does the same job with simpler usage:
```suggestion
oras login prod.registry.io --resolve prod.registry.io:5000:127.0.0.1 --plain-http
```
Also this file is auto-generated from `oras login -h` and should be modified on its own. `--resolve` is a common flag for all ORAS CLI commands targeting registry developers and it's better to have a blog to introduce it. @amands98 Are you interested in adding the blog?

Comment (amands98) [reply docs/commands/oras_login.mdx]: yeah, I can

File: docs/commands/oras_login.mdx [hunk 2]
```diff
@@ -49,6 +49,12 @@ Log in with username and password in an interactive terminal and no TLS check:
 oras login --insecure localhost:5000
 ```
 
+Log in to registry with customized DNS resolution
```
Comment (TerryHowe) [inline docs/commands/oras_login.mdx:52]:
```suggestion
Log in to a registry with custom DNS resolution
```

**Review by TerryHowe — APPROVED** [review]
/lgtm

File: docs/commands/oras_login.mdx (hunk 1 above)
Comment (FeynmanZhou) [reply docs/commands/oras_login.mdx]: @amands98 Would you mind creating an issue to track this blog item?

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Pull request merged**